src/databases/repositories/vipRepository.ts

```diff
@@ -0,0 +1,49 @@
+import { HashedUserID } from "../../types/user.model";
+import { db } from "../databases";
+import { IDatabase } from "../IDatabase";
+import { VipUser } from "../models";
+import { PrivateTempVipLog } from "../models/private/tempVipLog";
+
+export class VipRepository {
+    public static async addVip(hashedUserId: HashedUserID, publicDb: IDatabase): Promise<void> {
+        const vipUser = new VipUser({ userID: hashedUserId });
+
+        await publicDb.prepare("run", 'INSERT INTO "vipUsers" VALUES(?)', [vipUser.userID]);
+    }
```

Comment:
If these are all going to be static functions, I think getting rid of the class and just exporting the function itself makes more sense